src/clients/HubPoolClient.ts

```diff
@@ -244,29 +253,36 @@ export class HubPoolClient extends BaseAbstractClient {
   }
 
   async computeRealizedLpFeePct(
-    deposit: { quoteTimestamp: number; amount: BigNumber; destinationChainId: number; originChainId: number },
+    deposit: {
+      quoteTimestamp: number;
+      amount: BigNumber;
+      destinationChainId: number;
+      originChainId: number;
+      blockNumber: number;
+    },
```

Comment:
Should this be made into a type?

Reply:
Can we somehow tell Typescript that we only want to pluck these parts out of the `Deposit` type?

Reply:
Yep! We can use the `Pick<T, k typeof keyof T>`. It would be something like:

```
deposit: Pick<Deposit, "quoteTimestamp" | "amount" | "destinationChainId" | "blockNumber">
```